### PR TITLE
Update for Elm 0.19

### LINKED ISSUE #275: Upgrade to Elm 0.19
Hi! I'm not sure if this is already being done by someone, but if not I'd love to help upgrading _create-elm-app_ to Elm 0.19 – just let me know. :)

### PR DESCRIPTION
**Pull request by xtian**
Fixes #275

### CODE REVIEW

File: config/paths.js [hunk 1]
```diff
@@ -46,8 +46,8 @@ module.exports = {
   dotenv: resolveApp('./.env'),
   entry: resolveApp('./src/index.js'),
   appBuild: resolveApp('./build'),
-  elmPackageJson: resolveApp('./elm-package.json'),
-  elmMake: require('elm/platform').executablePaths['elm-make'],
-  publicUrl: getPublicUrl(resolveApp('elm-package.json')),
-  servedPath: getServedPath(resolveApp('elm-package.json'))
+  elmPackageJson: resolveApp('./elm.json'),
+  elm: '/usr/local/bin/elm',
```
Comment (xtian) [inline config/paths.js]: Should be updated when `elm/platform` has been released

**Comment by halfzebra** [comment]
Good initiative, looking forward to the public release of 0.19 👍

**Comment by Ealhad** [comment]
Released!
https://github.com/elm/compiler/blob/master/upgrade-docs/0.19

File: config/paths.js [hunk 2]
```diff
@@ -46,8 +46,8 @@ module.exports = {
   dotenv: resolveApp('./.env'),
   entry: resolveApp('./src/index.js'),
   appBuild: resolveApp('./build'),
-  elmPackageJson: resolveApp('./elm-package.json'),
-  elmMake: require('elm/platform').executablePaths['elm-make'],
-  publicUrl: getPublicUrl(resolveApp('elm-package.json')),
-  servedPath: getServedPath(resolveApp('elm-package.json'))
+  elmPackageJson: resolveApp('./elm.json'),
+  elm: require('elm/platform').executablePaths['elm'],
```
Comment (xtian) [inline config/paths.js]: It doesn't look like `executablePaths` is available with the new NPM package. Not sure how best to handle this.

Comment (halfzebra) [reply config/paths.js]: First places to check are https://github.com/rtfeldman/node-elm-compiler and https://github.com/elm-community/elm-webpack-loader
I'll be looking into this a bit later today.

Comment (xtian) [reply config/paths.js]: Yeah node-elm-compiler hasn't been updated recently and my PR to update elm-webpack-loader is https://github.com/elm-community/elm-webpack-loader/pull/142

**Comment by halfzebra** [comment]
Merging this into `release` branch to continue the preparation for the major release.

Thanks for the PR @xtian

**Pull request merged**